cmd/operator/main.go

```diff
@@ -133,6 +134,8 @@ func run(stop <-chan struct{}) {
 		logrus.Fatalf("invalid operator config: %v", err)
 	}
 
+	go periodicFullGC(cfg.KubeCli, cfg.MasterHost, cfg.Namespace, 10*time.Minute)
```

Comment:
probably we can make gc interval configurable? we also should write an e2e test to make sure it works.

create some services with etcd:app label set. check if operator will remove them within 2 * gc interval.